### PR TITLE
Add smaeul and myself as skalibs/execline/s6 maintainers

### PR DESCRIPTION
**Pull request by atlaua**
We maintain a few related ebuilds and have been taking care of these three packages for a while now, so let's make it official.

@smaeul has already acknowledged this, but we'll obviously need permission from @williamh too (though I think I discussed the matter with him on IRC earlier this year.)

### CODE REVIEW

**Comment by atlaua** [comment]
I contacted williamh via mail.

File: dev-lang/execline/metadata.xml [hunk 1]
```diff
@@ -5,6 +5,14 @@
 		<email>williamh@gentoo.org</email>
 		<name>William Hubbs</name>
 	</maintainer>
+	<maintainer type="person">
```
Comment (Zlogene) [inline dev-lang/execline/metadata.xml]: you should list gentoo-proxy-mmaintainers project last

**Comment by atlaua** [comment]
I've pushed a new version which adds proxy-maint everywhere, and also myself to @smaeul's s6-dns/s6-networking packages (I didn't even realize I wasn't on there already...).

**Comment by williamh** [comment]
Can we hold off on this? I realize I haven't been around much, mostly because of a very large project at work, but let me check the other p/r's first to see what the status is and catch back up.

**Comment by atlaua** [comment]
Sure, I'm not in a hurry with this. It'd make a lot of sense though,
smaeul and me have been handling all updates for nearly two years now.

Cheers,
Luis

**Comment by mgorny** [comment]
As requested on IRC, I'll merge this. Thanks!

**Pull request closed without merging**